claat/render/template-qwiklabs-git.md

```diff
@@ -0,0 +1,10 @@
+# {{.Meta.Title}}
```

Comment:
We should probably double check this high level layout with Casey before we actually do the export run. This is our best opportunity to change the "best practice" structure of our labs before its gets codified. For instance, so we actually want the feedback on the bottom.

(You very well might have already done this with Casey.)

No need to finalize it for this PR.